packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -969,6 +967,12 @@ if (hamburgerMenuContainer) {
       "iframe.goog-te-menu-frame.skiptranslate"
     );
 
+    // Hide "+" symbol from screen readers.
+    let hamburgerGTSymbol = utilNarrowNav.querySelector(".ma__utility-nav__translate #google_translate_element a span");
+    if (hamburgerGTSymbol) {
+      hamburgerGTSymbol.setAttribute("aria-hidden", "true");
```

Comment:
![Screen Shot 2023-11-22 at 12 40 13 PM](https://github.com/massgov/mayflower/assets/5789411/75ee9dd6-c9c3-4111-aabd-253311912ba5)

Weird, I'm still hearing the plus...

Reply:
ok so I gather that the UAT link is not always reliable https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-24290_navigation-labels/?p=pages-Homepage

I pulled this down locally and tested it again, now the phrase select language is not being announced, because it's wrapped inside of the aria-hidden span
![Screen Shot 2023-11-22 at 12 48 15 PM](https://github.com/massgov/mayflower/assets/5789411/dc5983a1-4664-414e-9d30-b68a08e64dc8)

Reply:
As moving the `:after` from `a` to `a > span`, the pseudo si  allied to all `span`s under `a`.  This was causing  this issue although strangely, all screen readers announced "select language" as I tested.

To resolve this, targeted the last `span` under `a` to have `aria-hidden` and the pseudo-class to add the symbol.

Reply:
`aria-label` is added to the UL. It's confirmed that JAWS and NVDA announced it.